.gitignore

```diff
@@ -17,3 +17,6 @@ packages/debugger/.tmp
 
 # @truffle/contract
 packages/contract/dist
+
+# @truffle/db
+**/*.db
```

Comment:
Heh, it's kind of unfortunate that this is necessary :)

Reply:
This will be interesting. We'd want to encourage backups and such, but git repositories don't seem like the correct mechanism to use.